src/main/groovy/life/qbic/samplestatus/reporter/api/UpdateSearchService.groovy

```diff
@@ -0,0 +1,29 @@
+package life.qbic.samplestatus.reporter.api
+
+import java.time.Instant
+
+/**
+ * <b>Interface UpdateSearchService</b>
+ *
+ * <p>Provides the time point of the last update search</p>
+ *
+ * @since 0.1.0
+ */
+interface UpdateSearchService {
+
+    /**
+     * <p>Returns the last update search time point.</p>
+     * @return the last update search time point.
+     *
+     * @since 0.1.0
+     */
+    Instant getLastUpdateSearchTimePoint()
```

Comment:
As this method is not needed yet, I would leave it out for now (or make it private) until it is needed.

Reply:
ah, its the necessity for @wow-such-code 's work. we need this for the query.